default/stringtables/en.txt

```diff
@@ -5868,25 +5886,26 @@ There are several Field Types, each having a distinctive, somewhat diffuse, appe
 ##
 
 GENERAL_PEDIA_REFERENCE
-Information such as this is accessible via the "Pedia" [[encyclopedia ENC_INDEX]].
+Information about game concepts, game objects, and the user interface is available from the "Pedia" [[encyclopedia ENC_INDEX]].
 
 DETAILED_PEDIA_REFERENCE
-[[GENERAL_PEDIA_REFERENCE]]  The Pedia can be opened and closed via the purple icon bearing a question mark, in the upper right portion of the main [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]]. The Pedia can be navigated via hotlinks and via the arrows in the lower right corner of the Pedia window.
+[[GENERAL_PEDIA_REFERENCE]]  The Pedia can be opened and closed via the purple icon bearing a question mark, in the upper right portion of the main [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]]. The Pedia can be navigated via hotlinks and via the arrows in the lower right corner of the Pedia window. When the Pedia window hides parts of other windows that you want to look at, simply close it. Opening it again will always return to the article that you last looked at.
 
 GREETINGS_GUIDE_TITLE
 **Greetings**
 
 GREETINGS_INTRO
 Greetings Esteemed Entity, and Welcome to the FreeOrion Universe!
 
-GREETINGS_GUIDE_REFERENCE
-Newcomers to this universe may particularly wish to consult the [[encyclopedia QUICK_START_GUIDE_TITLE]].
-
 GREETINGS_GUIDE_TEXT
 '''[[GREETINGS_INTRO]]
 
-[[DETAILED_PEDIA_REFERENCE]]
-[[GREETINGS_GUIDE_REFERENCE]]
+You may be as surprised as we are, but you have just been elected Emperor of our new-founded empire. You certainly can't wait to command our fleet of spaceships, to direct our research, and to govern our planets... Oh well, admittedly our one and only planet so far, but that's likely among the things you came here to change, isn't it?
+
+For guidance what to do first, please continue to the tutorial article [[encyclopedia TUTORIAL_RESEARCH_INTRO_TITLE]].
```

Comment:
Avoid "please"

Reply:
Changed in commit da06f7e0.

I'm no longer aware of any aspects that have to be fixed in the context of this PR, so maybe it can be pulled now.  Let me know if i missed anything that still needs fixing before pulling.